core/src/main/java/com/linecorp/armeria/server/HttpServerPipelineConfigurator.java

```diff
@@ -461,7 +471,8 @@ private void addHttpHandlers(ChannelHandlerContext ctx) {
                     config.http1MaxHeaderSize(),
                     config.http1MaxChunkSize()));
             p.addLast(new Http1RequestDecoder(config, ch, SCHEME_HTTPS, writer));
-            p.addLast(new HttpServerHandler(config, gracefulShutdownSupport, writer, H1, proxiedAddresses));
+            p.addLast(new HttpServerHandler(configHolder,
+                    gracefulShutdownSupport, writer, H1, proxiedAddresses));
```

Comment:
```suggestion
            p.addLast(new HttpServerHandler(configHolder,
                                            gracefulShutdownSupport,
                                            writer, H1, proxiedAddresses));
```

Reply:
Thanks @amitvc. I feel like we should really adopt an auto-formatter so we don't worry about this.

Reply:
+1 on this. I typically select the code and use intelliji to reformat the code. Code -> Reformat code option on intellij.